library/prolog_breakpoints.pl

```diff
@@ -61,18 +61,25 @@
 */
 
 %!  set_breakpoint(+File, +Line, +Char, -Id) is det.
+%!  set_breakpoint(+File, +Line, +Char, :Cond, -Id) is det.
```

Comment:
Does this need a meta_predicate directive?

Reply:
Added in https://github.com/SWI-Prolog/swipl-devel/pull/991/commits/03a3e5fb542b44d37d583be72c10c1c9736adc90